plugins/chat/app/serializers/chat_channel_serializer.rb

```diff
@@ -93,6 +94,10 @@ def current_user_membership
     ).as_json
   end
 
+  def last_message
+    ChatMessage.where('chat_channel_id': object.id).last
```

Comment:
This is probably already the case, but maybe we want to be explicit here we are not pulling a deleted message